src/main/scripts/src/app/shared/table/static-table/static-table.component.html

```diff
@@ -0,0 +1,25 @@
+<div>
```

Comment:
Why don't we use `<table>` here? And we should set `border-collapse: collapse` on it.